src/sentry/consumers/__init__.py

```diff
@@ -396,12 +415,43 @@ def build_consumer_config(group_id: str):
             healthcheck_file_path, strategy_factory
         )
 
+    if enable_dlq:
+        try:
+            dlq_topic = consumer_definition["dlq_topic"]
+        except KeyError as e:
+            raise click.BadParameter(
+                f"Cannot enable DLQ for consumer: {consumer_name}, DLQ topic is not specified in consumer definition"
+            ) from e
+        try:
+            cluster_setting = get_topic_definition(dlq_topic)["cluster"]
+        except ValueError as e:
+            raise click.BadParameter(
+                f"Cannot enable DLQ for consumer: {consumer_name}, DLQ topic {dlq_topic} is not configured in this environment"
+            ) from e
+
+        producer_config = get_kafka_producer_cluster_options(cluster_setting)
+        dlq_producer = KafkaProducer(producer_config)
+
+        dlq_policy = DlqPolicy(
+            KafkaDlqProducer(dlq_producer, Topic(dlq_topic)),
+            DlqLimit(
+                max_invalid_ratio=consumer_definition["dlq_max_invalid_ratio"],
+                max_consecutive_count=consumer_definition["dlq_max_consecutive_count"],
+            ),
+            None,
+        )
+
+        logger.info(f"DLQ enabled for {consumer_name} on {dlq_topic}")
```

Comment:
I'd remove this line, feels a bit weird that small detail is the only thing we'd log when starting a consumers

Reply:
Sure